S32-list/create.t

```diff
@@ -22,7 +22,17 @@ isa-ok($list_obj, List, 'Creating a new list object with new works.');
 is($list_obj, list(4, 5, 6), 'The list object contains the right values.');
 is(+$list_obj, 3, 'Finding the length functions properly.');
 }
-
+{
+my @list-pos is List = 1, 2, 3;
+isa-ok(@list-pos, List, "List can be created by 'is List'");
+isa-ok(@list-pos, Positional, 'Is Positional');
+is(@list-pos, (1, 2, 3), 'The list objects contains the right values.');
+}
+{
+my @list is List = 'foo';
+is(@list, ('foo',), 'The list constain a single element');
+is(+@list, 1, 'The lenght is one');
```

Comment:
Just a minor note on test logic and descriptions. The primary purpose of the test is to ensure the list is getting a single element. So, it makes sense to test for the number of elements first. And then test and explain if the element value what we expect it to be, not that it's just "a single element".